### PR TITLE
change "var" to "let"

### PR DESCRIPTION
**Pull request by naoyashiga**
I think we can change these "var" statement to "let". 

Please review.

### CODE REVIEW

File: DynamicColor/UIColor.swift [hunk 1]
```diff
@@ -153,7 +153,7 @@ public extension UIColor {
   :returns: A lighter UIColor.
   */
   public func lightenColor(amount: CGFloat) -> UIColor {
-    var amount = min(1, max(0, amount))
+    let amount = min(1, max(0, amount))
```
Comment (naoyashiga) [inline DynamicColor/UIColor.swift]: Variable( you declared) and internal parameter are same.

```swift
let _amount = 
```
It may better to understand.

**Comment by yannickl** [comment]
I pointed the problem before and I have forgotten to fix that, so thank you for your vigilance.

**Comment by naoyashiga** [comment]
Thank you for review!

File: DynamicColor/UIColor.swift (hunk 1 above)
Comment (yannickl) [inline DynamicColor/UIColor.swift]: Or may be you could rename it `normalizedAmount` or something similar?

Comment (naoyashiga) [inline DynamicColor/UIColor.swift]: OK. I will rename.

Comment (yannickl) [inline DynamicColor/UIColor.swift]: Thank you, and I'll merge it then. :+1:

Comment (naoyashiga) [inline DynamicColor/UIColor.swift]:
```swift
let normalizedWeight = min(1, max(0, weight))
```
normalizedWeight or weight?

Comment (yannickl) [inline DynamicColor/UIColor.swift]: I think `normalizedWeight` is great. ;)

**Pull request merged**